<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrint_3"/>
   <addaction name="actionPrint_Preview"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionColor"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="actionLeft_2"/>
   <addaction name="actionCenter_2"/>
   <addaction name="actionRight_2"/>
   <addaction name="actionJustify"/>
   <addaction name="separator"/>
   <addaction name="actionCreate_Style_1"/>
   <addaction name="actionMargins"/>
   <addaction name="actionDefault"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_3"/>
    <addaction name="actionOpen_2"/>
    <addaction name="actionSave_2"/>
    <addaction name="separator"/>
    <addaction name="actionPrint_2"/>
    <addaction name="actionPrint_Preview_2"/>
    <addaction name="separator"/>
    <addaction name="actionExit_3"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy_2"/>
    <addaction name="actionCut_2"/>
    <addaction name="actionPaste_2"/>
    <addaction name="separator"/>
    <addaction name="actionUndo_2"/>
    <addaction name="actionRedo_2"/>
    <addaction name="actionImage"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionFont_2"/>
    <addaction name="actionColor_2"/>
   </widget>
   <widget class="QMenu" name="menuStyle">
    <property name="title">
     <string>Style</string>
    </property>
    <addaction name="actionBold_2"/>
    <addaction name="actionItalic_2"/>
    <addaction name="actionUnderline_2"/>
    <addaction name="separator"/>
    <addaction name="actionLeft_4"/>
    <addaction name="actionCenter_4"/>
    <addaction name="actionRighy"/>
    <addaction name="actionJustify_2"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_Style"/>
    <addaction name="actionMargins_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuStyle"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/windows-copy-icon-28.png</normaloff>:/icons/image/windows-copy-icon-28.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/open.png</normaloff>:/icons/image/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/save.png</normaloff>:/icons/image/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/folder.png</normaloff>:/icons/image/folder.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/copy.png</normaloff>:/icons/image/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/cutting.png</normaloff>:/icons/image/cutting.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/files.png</normaloff>:/icons/image/files.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/undo.png</normaloff>:/icons/image/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/redo.png</normaloff>:/icons/image/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/font.png</normaloff>:/icons/image/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/eye-dropper.png</normaloff>:/icons/image/eye-dropper.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/bold.png</normaloff>:/icons/image/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/italic-font.png</normaloff>:/icons/image/italic-font.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="text">
    <string>Center</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionCreate_Style_1">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/icons8-selective-highlighting-24.png</normaloff>:/icons/image/icons8-selective-highlighting-24.png</iconset>
   </property>
   <property name="text">
    <string>Create Style</string>
   </property>
  </action>
  <action name="actionPrint_3">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/printer.png</normaloff>:/icons/image/printer.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/file.png</normaloff>:/icons/image/file.png</iconset>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionExport_PDF">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/icons8-export-pdf-40.png</normaloff>:/icons/image/icons8-export-pdf-40.png</iconset>
   </property>
   <property name="text">
    <string>Export PDF</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/Delete-80_icon-icons.com_57340.png</normaloff>:/icons/image/Delete-80_icon-icons.com_57340.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/underline.png</normaloff>:/icons/image/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionLeft_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/align-left.png</normaloff>:/icons/image/align-left.png</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionCenter_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/align_text_center_icon_155836.png</normaloff>:/icons/image/align_text_center_icon_155836.png</iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
  </action>
  <action name="actionRight_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/align-right.png</normaloff>:/icons/image/align-right.png</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionJustify">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/align.png</normaloff>:/icons/image/align.png</iconset>
   </property>
   <property name="text">
    <string>Justify</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionNew_3">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/windows-copy-icon-28.png</normaloff>:/icons/image/windows-copy-icon-28.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/open.png</normaloff>:/icons/image/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/save.png</normaloff>:/icons/image/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/folder.png</normaloff>:/icons/image/folder.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionPrint_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/printer.png</normaloff>:/icons/image/printer.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionPrint_Preview_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/file.png</normaloff>:/icons/image/file.png</iconset>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionExport_PDF_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/icons8-export-pdf-40.png</normaloff>:/icons/image/icons8-export-pdf-40.png</iconset>
   </property>
   <property name="text">
    <string>Export PDF</string>
   </property>
  </action>
  <action name="actionExit_3">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/Delete-80_icon-icons.com_57340.png</normaloff>:/icons/image/Delete-80_icon-icons.com_57340.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/copy.png</normaloff>:/icons/image/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/cutting.png</normaloff>:/icons/image/cutting.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/files.png</normaloff>:/icons/image/files.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/undo.png</normaloff>:/icons/image/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/redo.png</normaloff>:/icons/image/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionFont_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/font.png</normaloff>:/icons/image/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionColor_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/eye-dropper.png</normaloff>:/icons/image/eye-dropper.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionBold_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/bold.png</normaloff>:/icons/image/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/italic-font.png</normaloff>:/icons/image/italic-font.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionLeft_3">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionCenter_3">
   <property name="text">
    <string>Center</string>
   </property>
  </action>
  <action name="actionRight_3">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionUnderline_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/underline.png</normaloff>:/icons/image/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionLeft_4">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/align-left.png</normaloff>:/icons/image/align-left.png</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionCenter_4">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/align_text_center_icon_155836.png</normaloff>:/icons/image/align_text_center_icon_155836.png</iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
  </action>
  <action name="actionRighy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/align-right.png</normaloff>:/icons/image/align-right.png</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionJustify_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/align.png</normaloff>:/icons/image/align.png</iconset>
   </property>
   <property name="text">
    <string>Justify</string>
   </property>
  </action>
  <action name="actionText_Highlight_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/icons8-selective-highlighting-24.png</normaloff>:/icons/image/icons8-selective-highlighting-24.png</iconset>
   </property>
   <property name="text">
    <string>Text Highlight</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/icons8-добавить-изображение-16.png</normaloff>:/icons/image/icons8-добавить-изображение-16.png</iconset>
   </property>
   <property name="text">
    <string>Insert Image</string>
   </property>
  </action>
  <action name="actionCreate_Style">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/icons8-selective-highlighting-24.png</normaloff>:/icons/image/icons8-selective-highlighting-24.png</iconset>
   </property>
   <property name="text">
    <string>Create Style</string>
   </property>
  </action>
  <action name="actionMargins">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/icons8-отступ-48.png</normaloff>:/icons/image/icons8-отступ-48.png</iconset>
   </property>
   <property name="text">
    <string>Margins</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionMargins_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/image/icons8-отступ-48.png</normaloff>:/icons/image/icons8-отступ-48.png</iconset>
   </property>
   <property name="text">
    <string>Margins</string>
   </property>
  </action>
  <action name="actionNext_Page">
   <property name="text">
    <string>next</string>
   </property>
  </action>
  <action name="actionPrevious_Page">
   <property name="text">
    <string>previos</string>
   </property>
  </action>
  <action name="actiongo">
   <property name="text">
    <string>go</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
